test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/probe/LivenessProbeIntegrationTest.java

```diff
@@ -24,13 +24,18 @@
 import org.kie.cloud.api.deployment.Instance;
 import org.kie.cloud.api.deployment.KieServerDeployment;
 import org.kie.cloud.api.deployment.WorkbenchDeployment;
-import org.kie.cloud.api.scenario.WorkbenchKieServerDatabaseScenario;
+import org.kie.cloud.api.scenario.WorkbenchKieServerScenario;
 import org.kie.cloud.integrationtests.AbstractCloudIntegrationTest;
 import org.kie.cloud.integrationtests.util.TimeUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class LivenessProbeIntegrationTest extends AbstractCloudIntegrationTest<WorkbenchKieServerDatabaseScenario> {
+public class LivenessProbeIntegrationTest extends AbstractCloudIntegrationTest<WorkbenchKieServerScenario> {
```

Comment:
Liveness probe should check deployments in all scenarios, please consider making this test parametrized.